fjord/__init__.py

```diff
@@ -1,7 +1,16 @@
+# First we import absolute imports from the future, so that our celery.py
+# module will not clash with the library
+from __future__ import absolute_import
+
 import os
 import site
 import sys
 
+from fjord.manage_utils import monkeypatch
```

Comment:
ugh. so celery needs the monkeypatching to be in place too. Should we just move all the manage_utils stuff here?